src/views/TradingCompetition/components/Countdown/Timer.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
```

Comment:
A PR has been made exclusively for this component & the Timer tooltip - https://github.com/pancakeswap/pancake-frontend/pull/772